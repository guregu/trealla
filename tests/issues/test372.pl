:-initialization(main).
:-set_prolog_flag(public_clauses,on).

answer(41) :- abc, fail.
answer(42).

main :- H = answer(_), clause(H, Body), write_term((H :- Body), [nl(true), fullstop(true)]), fail.
main :- current_prolog_flag(public_clauses, Flag), writeln(Flag), fail.
main.
