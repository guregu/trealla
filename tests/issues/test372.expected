answer(41):-abc,fail.
answer(42):-true.
on
